R/f.R

```diff
@@ -52,7 +52,8 @@ f_rhs <- function(f) {
 #' @rdname f_rhs
 `f_rhs<-` <- function(x, value) {
   stopifnot(is_formula(x))
-  new_f(value, f_lhs(x), f_env(x))
+  f <- new_f(value, f_env(x), f_lhs(x))
+  maybe_pattern(f, x)
```

Comment:
this is copying the call name, e.g. either `~` or `:=`. I'll rename to `copy_call_name()`.